modules/EvseManager/EvseManager.cpp

```diff
@@ -29,6 +29,22 @@ void EvseManager::init() {
     if (!(config.ac_hlc_enabled || config.ac_with_soc || config.charge_mode == "DC")) {
         slac_enabled = false;
     }
+
+    // Use SLAC MAC address for Autocharge if configured.
+    if (config.autocharge_use_slac_instead_of_hlc && slac_enabled) {
+        r_slac[0]->subscribe_ev_mac_address([this](const std::string& _token) {
+            types::authorization::ProvidedIdToken autocharge_token;
+
+            std::string token = _token;
+            token.erase(remove(token.begin(), token.end(), ':'), token.end());
```

Comment:
this line is really hard to read, I would rename this function into something like
inline static void trim_colons_from_string(std::string& text) { ... } and also reuse it in its second occurence